Twig/Extension.php

```diff
@@ -176,4 +186,76 @@ public function isRtl($lang = false): bool
         }
         return $this->intuition->isRtl($this->intuition->getLang());
     }
+
+    /**
+     * Get the currently checked-out Git branch.
+     * @return string
+     */
+    public function gitBranch(): string
+    {
+        $process = new Process(['git rev-parse --symbolic-full-name --abbrev-ref HEAD']);
```

Comment:
I don't think this works, which is why I don't like the array format, despite what the docs say. I think it has to be something like `['git rev-parse', '--symbolic-full-name', '--abbrev-ref', 'HEAD']`

Reply:
Of course! hehe 

done